src/stores/entries.js

```diff
@@ -47,7 +47,7 @@ export const useEntryStore = defineStore('entries', {
         const entries = querySnapshot.docs.map((doc) => ({ id: doc.id, ...doc.data() }))
 
         for (const entry of entries) {
-          entry.author = userStore.getUserById(entry.author.id)
+          entry.author = userStore.getUserById(entry.author.id) ?userStore.getUserById(entry.author.id) :  entry.author
```

Comment:
It can be simplified:
```suggestion
          entry.author = userStore.getUserById(entry.author.id) ||  entry.author
```